peps/pep-0545.rst

```diff
@@ -3,8 +3,9 @@ Title: Python Documentation Translations
 Author: Julien Palard <julien@palard.fr>,
         Inada Naoki <songofacandy@gmail.com>,
         Victor Stinner <vstinner@python.org>
-Status: Final
+Status: Active
```

Comment:
This state change lists the PEP under "Process and Meta-PEPs" rather than "Historical Meta-PEPs and Informational PEPs"

Reply:
By coincidence, just 15 minutes earlier I suggested we should do this (at https://github.com/python/docsbuild-scripts/pull/225#discussion_r1820173291) :)